docs/source/DEVELOPER.rst

```diff
@@ -144,3 +144,29 @@ The Deepcell Kiosk uses Google Kubernetes Engine to requisition resources on Goo
 5. any Persistent Disks associated with your cluster
 
 While we hope this list is comprehensive, there could be some lingering resources used by Google Cloud and not deleted automatically that we're not aware of.
+
+Benchmarking the DeepCell Kiosk
+---------------------------------------------------------
+
+The DeepCell Kiosk comes with a utility for benchmarking the scalability and performance of a deep learning workflow. To run your own benchmarking, please read below.
+
+1. If you don't already have a cloud storage bucket devoted exclusively to DeepCell Kiosk benchmarking runs, you should create one now. It's fine to reuse this bucket for multiple benchmarking runs, but don't use it as the storage bucket for normal DeepCell Kiosk cluster operations.
```

Comment:
Why shouldn't they use the bucket for normal operations?

Reply:
If someone uploads something to the bucket during a benchmarking run, won't that mess things up? How do you think we should express that warning?

Reply:
It is an extra job that is being processed, but the benchmarking job will still be run.

Maybe just a warning that submitting extra jobs to a cluster during benchmarking will affect benchmarking performance.